src/main/java/io/nats/client/JetStream.java

```diff
@@ -13,15 +13,91 @@
 
 package io.nats.client;
 
+import io.nats.client.api.ConsumerConfiguration;
 import io.nats.client.api.PublishAck;
+import io.nats.client.api.StorageType;
+import io.nats.client.api.StreamConfiguration;
 import io.nats.client.impl.Headers;
 
 import java.io.IOException;
 import java.time.Duration;
 import java.util.concurrent.CompletableFuture;
 
 /**
- * JetStream context for creation and access to streams and consumers in NATS.
+ * JetStream context for access to streams and consumers in NATS.
+ * 
+ * <h3>Basic usage</h3>
+ * 
+ * <p>{@link #publish(String, byte[]) JetStream.Publish} will send a message on the specified subject as a Req/Rep waiting for acknowledgement. 
+ * An error will be received if no stream is litetning on said subject.
+ * 
+ * <p>{@link Connection#publish(String, byte[]) Connection.Publish} can be used as well to send to a stream on a subject, but no confirmation or error will come back. 
+ * Use non acknowledged publish with caution as it is easy to overwhelm Jetstream this way.
```

Comment:
"Use core publish..."

Not sure if you are aware of JetStreamOptions.publishNoAck which does a core publish and returns null for the PublishAck

Now that I said that, please remove the whole thing. Core publish to a stream is a footgun and we don't want to advertise it.

Reply:
Good point, removed, the whole idea is to show people the most common and correct usage patterns.